updates/pretty_info.json

```diff
@@ -11,6 +11,11 @@
       "max_schema_version": 9,
       "min_schema_version": 9
     },
+    "33": {
+      "changes": 0,
+      "max_schema_version": 11,
```

Comment:
is `max_schema_version` inclusive? WHat you wrote is `10 <= version < 11` or `10 <= version <= 11`

Reply:
> is max_schema_version inclusive?

yes, it is.

> WHat you wrote is 10 <= version < 11 or 10 <= version <= 11

it should be 9 <= version <= 10